backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -291,7 +305,11 @@ public Response deleteEndpoint(@Context SecurityContext sec, @PathParam("id") UU
         // Clean up the secrets in Sources.
         if (this.featureFlipper.isSourcesUsedAsSecretsBackend()) {
             Endpoint ep = endpointRepository.getEndpoint(orgId, id);
-            this.secretUtils.deleteSecretsForEndpoint(ep);
+            try {
+                this.secretUtils.deleteSecretsForEndpoint(ep);
+            } catch (final SourcesException e) {
+                Log.error(e.getMessage(), e);
+            }
```

Comment:
Here, we're deleting an endpoint and the Sources data deletion throws an exception. It looks fine to me to keep deleting the data on our side and log the exception. There is however a risk of leaving orphan records in the Sources DB, depending on the cause of the exception and which actions are taken by the dev team.

Reply:
I +1 to that, yes. I was thinking to add another `Info` line there, to say something like "[endpoint_uuid: %s] Integration removed with Sources errors. Associated secrets might not have been deleted" to easily see that "aha, this error message is tied to the successful deletion".

Reply:
Done!